ime/app/src/main/res/values/strings.xml

```diff
@@ -485,6 +485,16 @@
     <string name="bottom_generic_row_dialog_title">Select row type</string>
     <string name="bottom_generic_row_summary">Selected bottom row: %s</string>
 
+    <string name="popup_characters_order_title">Characters shown upon long press</string>
+    <string name="popup_characters_order_summary">Order is important. Uppercase letters represent different characters for every layout:\n
```

Comment:
How does this long explanation look in the UI? Can you share a screenshot?

Reply:
Sure, here. Please excuse the Spanglish.

![Screenshot_20210107-183045_AnySoftKeyboard_Dev](https://user-images.githubusercontent.com/37388405/103924712-560cbf00-510e-11eb-8a50-d18d09f64626.png)

I don't remember the exact details, but I had to tweak a template for an info section in XML. It must be at 380af81088c8494ef8917fda619f012027a6eaca (“Add better instructions for the characters reordering”).